lib/collection/item.js

```diff
@@ -179,6 +190,16 @@ _.assign(Item.prototype, /** @lends Item.prototype */ {
         return (requestAuth = extractAuth(this.request)) ? requestAuth : this.findInParents('auth', extractAuth);
     },
 
+    getProtocolProfileBehavior: function () {
+        var protocolProfileBehavior = extractProtocolProfileBehavior(this);
+
+        this.forEachParent({ withRoot: true }, function (entity) {
+            protocolProfileBehavior = Object.assign(extractProtocolProfileBehavior(entity), protocolProfileBehavior);
+        });
+
+        return protocolProfileBehavior;
+    },
+
```

Comment:
add jsdoc + add setter

Reply:
- get
-- not inherit

- set
-- set in place
-- override all below (destructive delete any conflicting behaviour in children)

Reply:
get without inheriting is just `entity.protocolProfileBehavior`